apple/Handlers/RNPanHandler.m

```diff
@@ -405,17 +449,21 @@ - (RNGestureHandlerEventExtraData *)eventExtraData:(NSPanGestureRecognizer *)rec
                                 withTranslation:[recognizer translationInView:recognizer.view.window.contentView]
                                    withVelocity:[recognizer velocityInView:recognizer.view.window.contentView]
                             withNumberOfTouches:1
-                                withPointerType:RNGestureHandlerMouse];
+                                withPointerType:RNGestureHandlerMouse
+                                 withStylusData:nil];
 }
 #else
 - (RNGestureHandlerEventExtraData *)eventExtraData:(UIPanGestureRecognizer *)recognizer
 {
+  RNBetterPanGestureRecognizer *panRecognizer = (RNBetterPanGestureRecognizer *)recognizer;
+
   return [RNGestureHandlerEventExtraData forPan:[recognizer locationInView:recognizer.view]
                            withAbsolutePosition:[recognizer locationInView:recognizer.view.window]
                                 withTranslation:[recognizer translationInView:recognizer.view.window]
                                    withVelocity:[recognizer velocityInView:recognizer.view.window]
                             withNumberOfTouches:recognizer.numberOfTouches
-                                withPointerType:_pointerType];
+                                withPointerType:_pointerType
+                                 withStylusData:[panRecognizer.stylusData toDictionary]];
```

Comment:
I know that technically this will work since objC will return nil when trying to invoke a method on a nil object but can it at least get a comment? It's very confusing for my little brain. This is the stupidest feature in a language.

Reply:
Added in [35ce0de](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/35ce0de7b8a8d8bfff860e8b13c16a690ef27324) 😅